### ISSUE TITLE
Nu doesn't suggest argument, but command after writing a command (tried on sys)

### ISSUE THREAD

**Issue by aaarkid** [nushell/nushell#6630]
### Describe the bug

Not only does Nu not suggest an argument to the command when the command has already been given, but it also takes too long as it tries to find every single file in the FS as an argument despite being unreasonable and extremely slow.

### How to reproduce

Write `sys | get`
Hit TAB
Wait.

### Expected behavior

Recommend me one of the field inside the returned struct of `sys` like:
host
cpu
disks
mem
net

### Screenshots

<img width="959" alt="image" src="https://user-images.githubusercontent.com/39987510/192658337-32b3154e-7d95-4b01-b433-af5194b13f00.png">

### Configuration

| key                | value                                 |
| ------------------ | ------------------------------------- |
| version            | 0.61.0                                |
| branch             |                                       |
| tag                |                                       |
| build_os           | linux-x86_64                          |
| rust_version       | rustc 1.59.0                          |
| cargo_version      | cargo 1.59.0                          |
| pkg_version        | 0.61.0                                |
| build_time         | 2022-04-12 20:38:15 +00:00            |
| build_rust_channel | release                               |
| features           | dataframe, default, trash, which, zip |
| installed_plugins  |                                       |

### Additional context

* Make suggestions for `get` better
* Stop suggesting the whole FS, more reasonable to only suggest files from the pwd when files are a reasonable suggestion
* Avoid showing all suggested commands if list is too big, and give a prompt to user so they choose to show all command intentionally, like bash asks you to confirm when there are too many possibilities

**Comment by aaarkid** [comment]
Other problems with suggestions include:
Suggestions permit on screen after typing on deleting characters: may look convenient, but this only shoots me in the foot most of the time because it rarely helps me giving suggestions in between typing checkpoint and it really hurts when deleting characters or finishing words with TAB enabled.
TAB should also do nothing if typed on a current empty word unless it shows the contents of the cwd as suggestions.

**Comment by aaarkid** [comment]
I really want to love this shell and make it my default, but suggestions really need to be fixed and be brought to at least a usable degree for users to have a reason to use this shell for more than a one-time hack.

**Comment by aaarkid** [comment]
I should note that tabbing takes too long in WSL only.

**Comment by sholderbach** [comment]
Yeah the file system accesses being unequally slow across the WSL2 filesystem boundary has caused several problems in nushell.

**Comment by sholderbach** [comment]
At this point it doesn't suggests commands anymore but still defaults to files/directories. This isn't as annoying but still not truly smart.
